setup.py

```diff
@@ -24,7 +24,10 @@
     print('Please install python3-pip and run setup.py again.')
     exit(-1)
 
-BASE_ENV = Path(os.environ.get('VIRTUAL_ENV', None)) or Path('/')
+if 'VIRTUAL_ENV' in os.environ:
+    BASE_ENV = Path(os.environ['VIRTUAL_ENV'])
+else:
+    BASE_ENV = Path('/')
```

Comment:
Suggestion for a more concise code:
`BASE_ENV = Path(os.environ.get('VIRTUAL_ENV', '/'))`